docs/main/faqs/faqs.md

```diff
@@ -62,6 +56,406 @@ Refer to the following links for more information on the backup and restore func
+openebs-lvm-node-4d94n     2/2     Running   0          5h28m
+openebs-lvm-node-gssh8     2/2     Running   0          5h28m
+openebs-lvm-node-twmx8     2/2     Running   0          5h28m
+```
+
+We can verify that key has been registered successfully with the LVM LocalPV CSI Driver by checking the CSI node object yaml :-
+
+```yaml
+$ kubectl get csinodes pawan-node-1 -oyaml
+apiVersion: storage.k8s.io/v1
+kind: CSINode
+metadata:
+  creationTimestamp: "2020-04-13T14:49:59Z"
+  name: k8s-node-1
+  ownerReferences:
+  - apiVersion: v1
+    kind: Node
+    name: k8s-node-1
+    uid: fe268f4b-d9a9-490a-a999-8cde20c4dadb
+  resourceVersion: "4586341"
+  selfLink: /apis/storage.k8s.io/v1/csinodes/k8s-node-1
+  uid: 522c2110-9d75-4bca-9879-098eb8b44e5d
+spec:
+  drivers:
+  - name: local.csi.openebs.io
+    nodeID: k8s-node-1
+    topologyKeys:
+    - openebs.io/nodename
+    - openebs.io/rack
+```
+
+We can see that "openebs.io/rack" is listed as topology key. Now we can create a storageclass with the topology key created :
+
+```yaml
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: openebs-lvmpv
+allowVolumeExpansion: true
+parameters:
+  volgroup: "lvmvg"
+provisioner: local.csi.openebs.io
+allowedTopologies:
+- matchLabelExpressions:
+  - key: openebs.io/rack
+    values:
+      - rack1
+```
+
+The LVM Local PV CSI driver will schedule the PV to the nodes where label "openebs.io/rack" is set to "rack1".
+
+:::note 
+If storageclass is using Immediate binding mode and storageclass allowedTopologies is not mentioned then all the nodes should be labeled using "ALLOWED_TOPOLOGIES" keys, that means, "ALLOWED_TOPOLOGIES" keys should be present on all nodes, nodes can have different values for those keys. If some nodes don't have those keys, then LVMPV's default scheduler can not effectively do the volume capacity based scheduling. Here, in this case the CSI provisioner will pick keys from any random node and then prepare the preferred topology list using the nodes which has those keys defined and LVMPV scheduler will schedule the PV among those nodes only.
+:::
+
+[Go to top](#top)
+
+### What is Local PV ZFS?
+
+Local PV ZFS is a CSI driver for dynamically provisioning a volume in ZFS storage. It also takes care of tearing down the volume from the ZFS storage once volume is deprovisioned.
+
+[Go to top](#top)
+
+### How to install Local PV ZFS?
+
+Make sure that all the nodes have zfsutils-linux installed. We should go to the each node of the cluster and install zfs utils
+
+```
+$ apt-get install zfsutils-linux
+```
+Go to each node and create the ZFS Pool, which will be used for provisioning the volumes. You can create the Pool of your choice, it can be striped, mirrored or raidz pool.
+
+Once ZFS POOL is created we can install OpenEBS ZFS driver by running the following command.
+
+```
+kubectl apply -f https://raw.githubusercontent.com/openebs/zfs-localpv/develop/deploy/zfs-operator.yaml
```

Comment:
@balaharish7 @datacore-gthomas this is the old way of installing. We need to update this part or provide a link to the main installation.

Reply:
removed this FAQ.